### ISSUE TITLE
Not respecting `defaultSeverity` value in tslint.json

### ISSUE THREAD

**Issue by RichiCoder1** [Microsoft/vscode-tslint#193]
Currently, when I configure `defaultSeverity` to warning, all tslint reported items are still reported as errors. If I configure individual item's seventies it works just fine.

My Config:
```json
{
  "extends": "tslint:latest",
  "rules": {
    "trailing-comma": {
      "severity": "default",
      "options": {"multiline": "never", "singleline": "never"}
    },
    "object-literal-sort-keys": false,
    "no-string-literal": false,
    "arrow-parens": false,
    "no-for-in-array": true,
    "no-invalid-this": true,
    "no-angle-bracket-type-assertion": false
  },
  "defaultSeverity": "warning"
}
```

**Comment by egamma** [comment]
@RichiCoder1 are you using the latest version 0.10.12 of vscode-tslint?

**Comment by RichiCoder1** [comment]
Do you mean `0.12.0`?

**Comment by egamma** [comment]
Sorry, vscode-tslint 0.12.0 and tslint 5. I cannot reproduce that the default severity is not honored, pls see the recording below

![severity](https://cloud.githubusercontent.com/assets/172399/25165893/ed77019a-24d8-11e7-900e-a76f46d11a7f.gif)

This is the project: https://github.com/Microsoft/vscode-tslint/tree/master/tslint-tests

**Comment by RichiCoder1** [comment]
Gif for scale:
![tslintbug gif](https://cloud.githubusercontent.com/assets/2391878/25237346/00330554-25b0-11e7-8b14-b5a721838817.gif)

What's weird is setting the severity on individual settings works, and is affected by the default serverity. That being said, I've run into other weird issues that I'm 90% certain are because I'm in a sub folder.

**Comment by egamma** [comment]
>That being said, I've run into other weird issues that I'm 90% certain are because I'm in a sub folder.

Can you repro this issues on the command line, then it can be narrowed down to a tslint problem. See all the vscode-tslint does it to interpret the severities it gets from tslint.

**Comment by pmlamotte** [comment]
I'm seeing this as well but it does seem to be coming from the tslint executable itself.

**Comment by testerez** [comment]
From tslint doc:
> This value is not inherited and is only applied to rules in this file.
https://palantir.github.io/tslint/usage/tslint-json/

Did you try to run tslint in a terminal to see if it outputs warnings as you expect?

**Comment by pamtbaau** [comment]
@testerez It seems `tslint` is working as pointed out by the tslint doc you refer to.

Running `tslint` in a terminal respects the `defaultSeverity` option in `tslint.json`, but only for rules mentioned in the `tslint.json` file. This means that all rules that need a change of severity level, have to be copied into `tslint.json` and switched on explicitely...

`vscode-tslint` by-the-way follows the output of `tslint` correctly.

As mentioned by @pmlamotte tslint is working on an [enhancement request](https://github.com/palantir/tslint/issues/2569).

**Comment by egamma** [comment]
@pamtbaau thanks for the clarification, closing this issue.
